Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml

```diff
@@ -0,0 +1,479 @@
+category: Authentication
+commonfields:
+  id: Microsoft_Graph_Files
+  version: -1
+configuration:
+- display: Server URL
+  name: host
+  required: true
+  type: 0
+- display: ID (received from the admin consent - see Detailed Instructions)
+  name: auth_id
+  required: true
+  type: 4
+- display: Token (received from the admin consent - see Detailed Instructions)
+  name: tenant_id
+  required: false
+  type: 0
+- display: Key (received from the admin consent - see Detailed Instructions)
+  name: enc_key
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Trust any certificate (insecure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy
+  name: proxy
+  required: false
+  type: 8
+description: 'Integration description: Microsoft Graph lets your app get authorized
+  access to files in OneDrive, SharePoint and MS Teams across all organization. (requires
+  admin consent).'
+display: Microsoft Graph Files
+name: Microsoft_Graph_Files
+script:
+  commands:
```

Comment:
general issue for all the yml file.
all of the following entries are missing description:
commands, commands arguments, and outputs.
please complete them.

Reply:
fixed